src/order/filter/n_ary.lean

```diff
@@ -154,7 +154,7 @@ le_inf (map₂_mono_right inf_le_left) (map₂_mono_right inf_le_right)
 lemma map₂_inf_subset_right : map₂ m f (g₁ ⊓ g₂) ≤ map₂ m f g₁ ⊓ map₂ m f g₂ :=
 le_inf (map₂_mono_left inf_le_left) (map₂_mono_left inf_le_right)
 
-@[simp] lemma map₂_pure_left : map₂ m (pure a) g = g.map (λ b, m a b) :=
+@[simp] lemma map₂_pure_left : map₂ m (pure a) g = g.map (m a) :=
```

Comment:
This was intentional, to avoid eta-reducing during simplification.

Reply:
Feel free to revert.

Reply:
```suggestion
@[simp] lemma map₂_pure_left : map₂ m (pure a) g = g.map (λ b, m a b) :=
```